mne/decoding/time_gen.py

```diff
@@ -362,27 +423,16 @@ def score(self, epochs=None, y=None):
                        [self.y_pred_[train] for train in split],
                        self.scorer_)
             for split in splits)
-
+        # XXX
         self.scores_ = [score for chunk in scores for score in chunk]
         return self.scores_
 
 
-def _predict_slices(X, estimators, cv, slices, predict_mode, predict_method):
-    """Aux function of GeneralizationAcrossTime that loops across chunks of
-    testing slices.
-    """
-    out = list()
-    for this_estimator, this_slice in zip(estimators, slices):
-        out.append(_predict_time_loop(X, this_estimator, cv, this_slice,
-                                      predict_mode, predict_method))
-    return out
-
-
```

Comment:
There was two distinct looping function (for train and for testing times) which is a bit difficult to parse. I merged them for clarity purposes.